src/timer/windows/SDL_systimer.c

```diff
@@ -103,31 +103,35 @@ Uint64 SDL_GetPerformanceFrequency(void)
     return (Uint64)frequency.QuadPart;
 }
 
-void SDL_SYS_DelayNS(Uint64 ns)
+typedef NTSTATUS (NTAPI *NtDelayExecution_t)(BOOLEAN Alertable, PLARGE_INTEGER DelayInterval);
```

Comment:
Trying this pr locally, `NSTATUS` is not defined.
Including `<ntdef.h> ` makes the typedef available.